flit-client/src/App.tsx

```diff
@@ -13,10 +13,10 @@ function App() {
     <Router>
       <div className="App">
         <Switch>
-          <Route path="/conversation">
+          <Route path="/conversation/:cid">
             <ConversationPage />
           </Route>
-          <Route path="/">
+          <Route exact path="/">
```

Comment:
If we make this exact, we probably want to also add some sort of a catch-all route, to avoid the situation of the app rendering the container with no content in it.